### PR TITLE
W-13672431-Excel-session-cv

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**

### CODE REVIEW

**Review by ms-tlu — APPROVED** [review]
LGTM 👍

File: rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-session.adoc [hunk 1]
```diff
@@ -2,27 +2,20 @@
...
-The _Excel Session_ is an element that encompasses all Excel operations.
+Excel Session is an element that encompasses all Excel operations.
 Excel operations include the reading, modification, and creation of
-entire files and file elements in the Excel format.
+entire files and file elements in the Excel format. Excel Session supports reading and writing password protected Excel files.
 
-To use Excel operations it is not necessary to have Microsoft Excel
-installed on the computer.
+In Excel Session you define which Excel files to open,
+their location, and how to use them. Use other Action Steps within the Excel Session to process the information from the Excel files.
```
Comment (glenn-rodgers-sf) [inline rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-session.adoc]:
```suggestion
their location, and how to use them. Use other Action Steps within Excel Session to process the information from the Excel files.

```

File: rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-session.adoc [hunk 2]
```diff
@@ -2,27 +2,20 @@
...
 
-In the Excel Session you define which Excel files should be opened,
-where these are located at what time, and how they should be used.
+Excel Session opens the Excel files at the beginning of the execution and closes them at the end of the session. An Excel Session may contain a maximum of one further Excel Session.
```
Comment (glenn-rodgers-sf) [inline rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-session.adoc]:
```suggestion
Excel Session opens the defined Excel files at the beginning of the execution and closes them at the end of the session. An Excel Session may contain a maximum of one further Excel Session.

```

File: rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-session.adoc [hunk 3]
```diff
@@ -2,27 +2,20 @@
...
 
-By placing other Action Steps within the _Excel Session_ it is possible
-to process the information from the Excel files further.
+To execute a loop across all the elements in a row or column in the Excel file, use the xref:toolbox-excel-operations-excel-controlled-loop.adoc[] Actions Step.
```
Comment (glenn-rodgers-sf) [inline rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-session.adoc]:
```suggestion
To execute a loop across all the elements in a row or column in an Excel file, use the xref:toolbox-excel-operations-excel-controlled-loop.adoc[] Actions Step. 

```

File: rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-session.adoc [hunk 4]
```diff
@@ -108,39 +98,23 @@ This Action Step has no outbound variables.
...
+Use the *Operation Mode* section to choose whether to only read the Excel file, read and modify, or read and then write to a different Excel file. 
 
-The *source file* is the file that should be read. You can write the path to the directory where the source file
-is located directly in the input field, insert Environment and Script Variables, make a selection using a file explorer
-//image:media\image1.png[image,width=20,height=20]
-or connect with a Pin Variable. You also have the option of using a different file at the time
-of execution to at the time of creation.
-//Simply select the checkbox image:media\image2.png[image,width=303,height=15].
+The *Source file* is the file to read. Specify the path to the directory where the source file
+is located directly in the input field, inserting Environment and Script Variables, selecting it using a file explorer, or by using a Pin Variable.
```
Comment (glenn-rodgers-sf) [inline rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-session.adoc]: This sentence doesn't really make sense to me. Do you specify the directory by inserting the environment?

File: rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-session.adoc (hunk 2 above)
Comment (glenn-rodgers-sf) [reply rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-session.adoc]: Maybe imbedded excel session instead of further?

**Pull request merged**